example/linux/Makefile

```diff
@@ -28,11 +28,15 @@ SOURCES=flutter_embedder_example.cc
 # (e.g., optimization level) of the C++ code.
 BUILD=debug
 
+# Settings required to be set via command line:
+#
+# FLUTTER_ARTIFACT_CACHE_DIR is required to be set via command line.
+# FLUTTER_ENGINE_VERSION_FILE is required to avoid uneccessary engine rebuilds.
+
 # Dependency locations
 FLUTTER_APP_DIR=$(CURDIR)/..
 FLUTTER_APP_BUILD_DIR=$(FLUTTER_APP_DIR)/build
 FLUTTER_ROOT:=$(shell cat $(CURDIR)/.generated_flutter_root)
-FLUTTER_ARTIFACT_CACHE_DIR=$(FLUTTER_ROOT)/bin/cache/artifacts/engine/linux-x64
```

Comment:
Why not just leave this as a fallback for if it's not set, rather than making it a required parameter?